packages/mjml-image/src/index.js

```diff
@@ -133,7 +133,7 @@ export default class MjImage extends BodyComponent {
   }
 
   headStyle = (breakpoint) => `
```

Comment:
Maybe we could pass once for all a mobileBreakpoint and desktopBreakpoint ? It changes a bit the API but at least it will be computed only once ?

Reply:
I'm unsure because of the naming, but I'll do this.

Reply:
Yeah might be better to have an object instead of only  breakpoint so it would be extensible in the future.
I don't know if a MJML5 release is near soon so :/